.github/workflows/build-simd.yml

```diff
@@ -0,0 +1,39 @@
+name: Create and publish simd Docker image
+
+on: workflow_dispatch
```

Comment:
should we make this push to master/main also can make it on pull_request to test in the PR

Reply:
I can add on pull_request and then remove before merging, ideally we'll actually just want this is the release task.